playgrounds/react-vite/src/tabs/Debug.tsx

```diff
@@ -1,270 +1,287 @@
+import { useState } from 'react';
+import {
+	type Hex,
+	decodeErrorResult,
+	decodeFunctionData,
+	toFunctionSelector,
+	trim,
+} from 'viem';
+import { useSwitchChain } from 'wagmi';
 
-import { SeaportABI } from "../lib/abis/seaport";
+import { SeaportABI } from '../lib/abis/seaport';
 
 const ABIs = {
-  ERC20: ERC20_ABI,
-  ERC721: ERC721_ABI,
-  ERC1155: ERC1155_ABI,
-  SequenceMarketplaceV1: SequenceMarketplaceV1_ABI,
-  SequenceMarketplaceV2: SequenceMarketplaceV2_ABI,
-  Seaport: SeaportABI,
+	ERC20: ERC20_ABI,
+	ERC721: ERC721_ABI,
+	ERC1155: ERC1155_ABI,
+	SequenceMarketplaceV1: SequenceMarketplaceV1_ABI,
+	SequenceMarketplaceV2: SequenceMarketplaceV2_ABI,
+	Seaport: SeaportABI,
 } as const;
 
 export function Debug() {
-  const [selectedAbi, setSelectedAbi] = useState<keyof typeof ABIs>("ERC20");
-  const [errorData, setErrorData] = useState<Hex>();
-  const [inputData, setInputData] = useState<Hex>();
-
-  const handleDecodeError = () => {
-    try {
-      const decoded = decodeErrorResult({
-        abi: ABIs[selectedAbi],
-        data: errorData as Hex,
-      });
-      console.dir(decoded)
-    } catch (err) {
-      console.dir(err)
-    }
-  };
-
-  const handleDecodeData = () => {
-    try {
-      const decoded = decodeFunctionData({
-        abi: ABIs[selectedAbi],
-        data: trim(inputData as Hex)
-      });
-      console.dir(decoded);
-    } catch (err) {
-      console.dir(err);
-    }
-  };
-
-  const getFunctionSignatures = () => {
-    const abi = ABIs[selectedAbi];
-    return abi
-      .filter((item) => item.type === 'function')
-      .map((func) => {
-        try {
-          const signature = toFunctionSelector(func)
-          return `${func.name}: ${signature}`;
-        } catch (err) {
-          return `${func.name}: Error generating signature`;
-        }
-      });
-  };
+	const [selectedAbi, setSelectedAbi] = useState<keyof typeof ABIs>('ERC20');
+	const [errorData, setErrorData] = useState<Hex>();
+	const [inputData, setInputData] = useState<Hex>();
 
-  const { switchChain } = useSwitchChain()
+	const handleDecodeError = () => {
+		try {
+			const decoded = decodeErrorResult({
+				abi: ABIs[selectedAbi],
+				data: errorData as Hex,
+			});
+			console.dir(decoded);
+		} catch (err) {
+			console.dir(err);
+		}
+	};
 
-  return (
-    <Box paddingTop="3" gap="3" flexDirection="column">
-      <Card>
+	const handleDecodeData = () => {
+		try {
+			const decoded = decodeFunctionData({
+				abi: ABIs[selectedAbi],
+				data: trim(inputData as Hex),
+			});
+			console.dir(decoded);
+		} catch (err) {
+			console.dir(err);
+		}
+	};
 
-        <Box justifyContent="space-between">
-          <Text variant="large">Debug Tools</Text>
+	const getFunctionSignatures = () => {
+		const abi = ABIs[selectedAbi];
+		return abi
+			.filter((item) => item.type === 'function')
+			.map((func) => {
+				try {
+					const signature = toFunctionSelector(func);
+					return `${func.name}: ${signature}`;
```

Comment:
```suggestion
					return `${func.type}: ${signature}`;
```

`func` doesn't seem to have a `name`.